glib/src/collections/strv.rs

```diff
@@ -11,6 +11,21 @@ use crate::{translate::*, GStr, GString};
 #[repr(transparent)]
 pub struct StrVItem(ptr::NonNull<c_char>);
 
+#[doc(hidden)]
+unsafe impl TransparentPtrType for StrVItem {}
+
+#[doc(hidden)]
+impl GlibPtrDefault for StrVItem {
+    type GlibType = *mut c_char;
+}
+
+impl Clone for StrVItem {
+    #[inline]
+    fn clone(&self) -> StrVItem {
+        unsafe { StrVItem(ptr::NonNull::new_unchecked(ffi::g_strdup(self.0.as_ptr()))) }
```

Comment:
You also need `Drop` then

Reply:
Might make sense to rename this and give it a constructor too. By having `Clone` we make it proper API :)

Reply:
`GStrPtr`?

Reply:
Ok... let me drop it from this PR and open a separate one